dev.bat

```diff
@@ -5,6 +5,8 @@ virtualenv %VENV% --always-copy
 if %errorlevel% neq 0 exit /b %errorlevel%
 call %VENV%\Scripts\activate.bat
 if %errorlevel% neq 0 exit /b %errorlevel%
+pip install -U pip
```

Comment:
That's a bad idea on Windows, you want `python -m pip install -U pip`. Otherwise, it can't overwrite the `pip.exe` executable as it's currently in use.